src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/SuccessUpdater.kt

```diff
@@ -1,12 +1,33 @@
 package uk.gov.justice.digital.hmpps.hmppstier.service
 
+import com.amazonaws.services.sns.AmazonSNSAsync
+import com.google.gson.Gson
+import org.springframework.beans.factory.annotation.Value
 import org.springframework.stereotype.Service
-import uk.gov.justice.digital.hmpps.hmppstier.client.CommunityApiClient
-import uk.gov.justice.digital.hmpps.hmppstier.dto.TierDto
+import java.util.UUID
 
 @Service
-class SuccessUpdater(val client: CommunityApiClient) {
-  fun update(tierDto: TierDto, crn: String) {
-    client.updateTier("${tierDto.protectLevel.value}${tierDto.changeLevel.value}", crn)
+class SuccessUpdater(
+  private val amazonSNS: AmazonSNSAsync,
+  @Value("\${hmpps-events.topic}")private val topic: String,
+  private val gson: Gson
+) {
+
+  fun update(crn: String, calculationId: UUID) {
+    val tierChangeEvent = TierChangeEvent(EventType.HMPPS_TIER_CALCULATION_COMPLETE, crn, calculationId)
+    val message = Message(gson.toJson(tierChangeEvent), calculationId.toString())
+    amazonSNS.publish(topic, gson.toJson(message))
   }
 }
+
+private data class TierChangeEvent(
+  val eventType: EventType?,
```

Comment:
isn't this always populated?

Reply:
Good spot, i've copied it the domain from the client project and there is it ? because its an enum of either 'ones we care about or null' if it was a String it would always be populated in the client too.